frontend/integration-tests/tests/kubevirt/utils/utils.ts

```diff
@@ -47,12 +40,33 @@ export function removeLeakedResources(leakedResources: Set<string>) {
   leakedResources.clear();
 }
 
+export function addLeakableResource(leakedResources: Set<string>, resource) {
+  leakedResources.add(
+    JSON.stringify({name: resource.metadata.name, namespace: resource.metadata.namespace, kind: resource.kind})
+  );
+}
+
+export function removeLeakableResource(leakedResources: Set<string>, resource) {
+  leakedResources.delete(
+    JSON.stringify({name: resource.metadata.name, namespace: resource.metadata.namespace, kind: resource.kind})
+  );
+}
+
+export function createResource(resource) {
+  execSync(`echo '${JSON.stringify(resource)}' | kubectl create -f -`);
+}
+
 export function createResources(resources) {
   resources.forEach(resource => {
```

Comment:
we can call createResource forEach resource

Reply:
Right

Reply:
no need to use lambda